source/slang/hlsl.meta.slang

```diff
@@ -10453,6 +10528,7 @@ int sign(T x)
     __target_switch
     {
     case hlsl: __intrinsic_asm "sign";
+    case metal: __intrinsic_asm "sign";
```

Comment:
The return type of `sign` is float.
You may need to type-cast to int like how glsl does one line below.